src/components/Banner/tests/Banner.test.tsx

```diff
@@ -0,0 +1,197 @@
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBe('Order archived');
+        });
+
+        it('should verify banner title property is not defined', () => {
+            const bannerWrapper = mount(<Banner>
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBeUndefined();
+        });
+
+        it('should verify banner status property', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" status="success" >
+                                        <p>This order was archived on March 7.</p> 
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('success');
+        });
+
+        it('should verify banner status property is not defined', () => {
+            const bannerWrapper = mount(<Banner>
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBeUndefined();
+        });
+
+        it('should verify banner icon property', () => {
+            const bannerWrapper = mount(<Banner status="success" icon="arrowDown">
+                                        <p>This order was archived on March 7.</p> 
+                                        </Banner>);
+            expect(bannerWrapper.prop('icon')).toBe('arrowDown');
+        });
+
+        it('should verify banner icon property is not defined', () => {
+            const bannerWrapper = mount(<Banner>
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBeUndefined();
+        });
+
+        it('should verify banner children property', () => {
+            const bannerWrapper = mount(<Banner
+                                        title="Some of your product variants are missing weights"
+                                        status="warning"
+                                        action={{content: 'Edit variant weights'}}>
+                                        <p>Add weights to show accurate rates.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('children').props.children).toBe('Add weights to show accurate rates.');
+        });
+
+        it('should verify banner children property is not defined', () => {
+            const bannerWrapper = mount(<Banner/>);
+            expect(bannerWrapper.prop('children')).toBeUndefined();
+        });
+    });
+
+    describe('Verify Banner component functions called', () => {
+        it('should verify banner action is called', () => {
+            const spy = jest.fn();
+            const action: Action = {content: 'Action Content', onAction: () => { spy(); } };
```

Comment:
Format object code as below. Apply this to all your tests in banner.
```

const action: Action = {
    content: 'Action Content', 
    onAction: spy, 
};
```
You can directly bind spy to onAction prop. Check it once by executing this test.